content/en/synthetics/troubleshooting/_index.md

```diff
@@ -30,7 +30,11 @@ If you see a sudden spike or overall increase in your API test [timing metrics][
 
 #### The website is not loading in the iframe
 
-After downloading the [Datadog extension][4], you are unable to see your website in the iframe on the right side of your Browser test's recorder and the iframe displays `Your website does not support being loaded through an iframe.`. This could mean that your application has some settings preventing it from being opened in an iframe. If that is the case, try opening your website in a pop-up by clicking **Open in Popup** to record your journey.
+After downloading the [Datadog extension][4], you are unable to see your website in the iframe on the right side of your Browser test's recorder and the iframe displays `Your website does not support being loaded through an iframe.`. This could mean that your application has some settings preventing it from being opened in an iframe. 
+
+Or, if you are unable to login to your website when recording in the iframe recorder, this could mean that your application has a request that is blocked when viewing the console.
```

Comment:
```suggestion
Or, if you are unable to login to your website when recording in the iframe recorder, this could mean that your application has a request that is blocked.
```